test/e2e/generator/generate.go

```diff
@@ -105,6 +105,23 @@ func Generate(r *rand.Rand, opts Options) ([]e2e.Manifest, error) {
 			continue
 		}
 
+		if opt["p2p"] == HybridP2PMode {
+			numLegacy := 0
+			for _, n := range manifest.Nodes {
+				if n.UseLegacyP2P {
+					numLegacy++
+				}
+			}
+			if numLegacy == len(manifest.Nodes) {
+				fmt.Println("omit")
+				continue
+			}
+			if numLegacy == 0 {
+				fmt.Println("omit", "omit")
```

Comment:
I'm not sure I follow what these prints are doing. Does this add to the output from the call to generateTestnet in some way?

Reply:
prints are an error